### ISSUE TITLE
Commit https://github.com/NixOS/nixpkgs/commit/8833983f261c6afa0361465f31c4dbc39c45b386 in nixpkgs overrides hydra-init.service's permissions on every NixOS Activation

### ISSUE THREAD

**Issue by AmandaCameron** [NixOS/hydra#966]
I'm not entirely sure, I suspect https://github.com/NixOS/nixpkgs/commit/8833983f261c6afa0361465f31c4dbc39c45b386 is responsible for some behaviour I've been seeing for a couple months now, where any time I do a NixOS Rebuild, I've got to manually re-start hydra-init.service to make sure the queue runner, etc, can actually do it's job.

**Comment by FlorianFranzen** [comment]
You might have also hit #366.

**Comment by AmandaCameron** [comment]
Nope, if I manually change the permissions as well it'll start working. Looking in the logsmentions access denied messages, IIRC
